src/driver/amdxdna/aie2_pci.h

```diff
@@ -257,7 +257,8 @@ struct amdxdna_dev_hdl {
 	struct async_events		*async_events;
 
 	u32				dev_status;
-	u32				ctx_num;
+	u32				hwctx_num;
```

Comment:
call it hwctx_cnt? it is consistent with ctx_cnt.

Reply:
done